.gitattributes

```diff
@@ -1,3 +1,2 @@
 * text eol=lf
-
-*.tgz binary
+*.tgz filter=lfs diff=lfs merge=lfs -text
```

Comment:
@zkochan `lfs` appears to not been removed here. Should we be worried?

Reply:
can you remove the first 2 commits in this PR?